esp/esp/tagdict/models.py

```diff
@@ -33,8 +38,25 @@ def __unicode__(self):
 
     EMPTY_TAG = " "
 
-    @cache_function
+    @classmethod
     def getTag(cls, key, target=None, default=None):
+        if key not in all_global_tags:
+            logger.warning("Tag %s not in list of global tags", key)
+        if target is not None:
+            logger.warning("getTag() called for key %s with specific target; consider using getProgramTag()",
+                           key)
+
+        result = cls._getTag(key, target=target)
+
+        if result and (result.lower() == "false" or result.lower() == "true"):
```

Comment:
Sure, except there's at least one case where `default` is `True` (not the string) so I'd have to write it a bit differently. That's also a thing I should try to catch, actually.